app/pages/BeelieversAuction/MintInfo.tsx

```diff
@@ -97,57 +131,60 @@ interface MintInfoProps {
 	user?: User;
 }
 
-export function MintInfo({ user, auctionInfo: { clearingPrice } }: MintInfoProps) {
+export function MintInfo({ user, auctionInfo: { clearingPrice, auctionSize } }: MintInfoProps) {
```

Comment:
if user === null, then we should return early and simplify the code below

Reply:
not all information below is dependent on user e.g minting price. we can't return early.